css/when_and_how_to_create_a_new_css_component.md

```diff
@@ -0,0 +1,75 @@
+# When and how to create a new CSS component?
+
+## What is a component:
+
+Following Atomic design principles; a component can be molecules or organism which can work alone or as a part of a page in combination with other components.
```

Comment:
A link would be great here (could be to http://bradfrost.com/blog/post/atomic-web-design/ or some other resource) the first time you mention "Atomic design principles".